proto/tpm_enrollz.proto

```diff
@@ -81,6 +83,25 @@ message RotateOIakCertResponse {
   // changes.
 }
 
+// The RotateAIKCertRequest handles the workflow for enrollment of TPM1.2
+// devices. The initial request will include the issuer_public_key to allow
+// the building of the AIK which will then be returned and used to generate
+// the AIK cert. For any updates after initial enrollment the new AIK cert
+// can just be returned.
+message RotateAIKCertRequest {
+  oneof value {
+    bytes issuer_public_key = 1;
+    string aik_cert = 2;
+  }
+}
+
+message RotateAIKCertResponse {
+  oneof value {
+    bytes application_identity_public_key = 1;
```

Comment:
This is basically an identity request, maybe name it as `application_identity_request` ?